main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -71,7 +73,11 @@ public ValidationRunner(VersionResolver versionResolver) {
     this.versionResolver = versionResolver;
   }
 
+  @MemoryMonitor
   public Status run(ValidationRunnerConfig config) {
+    MemoryUsageRegister.getInstance().clearRegistry();
+    var memoryBefore =
```

Comment:
Why do we need to do that here? I thought that because of @MemoryMonitor this would be done automatically in MemoryMonitorAspect.monitorMemoryUsage

Reply:
This is the case of concurrent threads. If we use the annotation, the AspectJ is called multiple times because multiple threads are created inside the method.

Reply:
Added a comment within the file and a section in the acceptance tests documentation.